### PR TITLE
Added Island Level Configuration

### PR DESCRIPTION
**Pull request by sh0inx**
- the other side of [IridiumTeams #49](https://github.com/Iridium-Development/IridiumTeams/pull/49)

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java [hunk 1]
```diff
@@ -56,6 +56,7 @@ public Configuration() {
     public boolean clearEnderChestOnRegen = false;
     public boolean allowPvPOnIslands = false;
     public boolean islandCreateOnJoin = false;
+    public boolean islandCreationCost = false;
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java:59]: I feel like this is going to confuse things even more. I understand there are alot of people asking this in the Discord, but if we want to remove the creation cost, I feel setting the default cost to 0 for everything is a better aproach

File: src/main/java/com/iridium/iridiumskyblock/gui/BiomeCategoryGUI.java [hunk 2]
```diff
@@ -61,7 +62,7 @@ public void addContent(Inventory inventory) {
 
     private List<Placeholder> getBiomeLorePlaceholders(Biomes.BiomeItem item) {
         List<Placeholder> placeholders = new ArrayList<>(Arrays.asList(
-                new Placeholder("amount", IridiumSkyblock.getInstance().getBiomeManager().formatPrice(item.defaultAmount)),
+                new Placeholder("minLevel", String.valueOf(item.minLevel)),
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/gui/BiomeCategoryGUI.java:65]: Do we not need the Amount placeholder anymore?

File: src/main/java/com/iridium/iridiumskyblock/managers/BiomeManager.java [hunk 3]
```diff
@@ -78,14 +75,40 @@ private void setBankBalance(Player player, String bankItem, double amount) {
     }
 
     private boolean canPurchase(Player player, Biomes.BiomeItem biomeItem) {
+
+        if(biomeItem.minLevel != 1) {
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/managers/BiomeManager.java:79]:
```suggestion
        if(biomeItem.minLevel > 1) {
```

File: src/main/java/com/iridium/iridiumskyblock/gui/BiomeCategoryGUI.java (hunk 2 above)
Comment (sh0inx) [reply src/main/java/com/iridium/iridiumskyblock/gui/BiomeCategoryGUI.java:65]: yes, this allows us to fetch the price of the schematic from the config directly without having to set the lore manually every time

File: src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java (hunk 1 above)
Comment (sh0inx) [reply src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java:59]: this was more for the fact that we could allow pricing for creation *and* regen to be different (as in, when you create the island, it doesn't cost anything, but when you regen, it does) - we could change it to be free but honestly i dont care about those people lol

**Review by PeachesMLG — APPROVED** [review]

**Pull request merged**